### PR TITLE
add getAll, find and changeName methods

### PR DESCRIPTION
**Pull request by Nyanko-sensei**
resolves #2107

### Changelist

- [ ] Added a method to show all tokens `get all`
- [ ] Added a method to show single api token info `find`
- [ ] Added a method to change token name `changeName`

### Testing/QA

- Open deployed branch

### How to release

- none extra steps

### PR Checklist

- [ ] Tests are done and pass
- [ ] No PHP and JS errors
- [ ] PR does not go out of scope of the task
- [ ] Possible breaking changes are documented in the PR

### Screenshots

not applicable

### Risks

None  at this point

### Additional notes

It goes with other PR in docs and php-sdk

### CODE REVIEW

**Review by fosron — APPROVED** [review]

File: src/Endpoints/Token.php [hunk 1]
```diff
@@ -13,6 +13,19 @@ class Token extends AbstractEndpoint
...
 
+    /**
+     * @return array
+     * @throws ClientExceptionInterface
+     * @throws JsonException
+     */
+    public function getAll(): array
+    {
+
+        return $this->httpLayer->get(
```
Comment (JoaoFSCruz) [inline src/Endpoints/Token.php:24]: This method is lacking the pagination parameters.

File: README.md [hunk 2]
```diff
@@ -52,7 +52,10 @@ MailerSend PHP SDK
         * [Get a single scheduled message](#get-a-single-scheduled-message)
         * [Delete a scheduled message](#delete-a-scheduled-message)
     * [Tokens API](#tokens)
+        * [Get all tokens](#get-all-tokens)
+        * [Find a tokens](#find-a-token )
```
Comment (JoaoFSCruz) [inline README.md:56]:
```suggestion
        * [Find a token](#find-a-token)
```

**Review by JoaoFSCruz — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -52,7 +52,10 @@ MailerSend PHP SDK
         * [Get a single scheduled message](#get-a-single-scheduled-message)
         * [Delete a scheduled message](#delete-a-scheduled-message)
     * [Tokens API](#tokens)
+        * [Get all tokens](#get-all-tokens)
+        * [Find a tokens](#find-a-token)
```
Comment (PeterOcansey) [inline README.md:56]: Change to `Find a token`

File: README.md [hunk 4]
```diff
@@ -1063,6 +1066,34 @@ $mailersend->scheduleMessages->delete('message_id');
 
 ## Tokens
 
+<a name="get_all_tokens"></a>
+
+### Get all tokens
+
+```php
+use MailerSend\MailerSend;
+use MailerSend\Helpers\Builder\TokenParams;
```
Comment (PeterOcansey) [inline README.md:1075]: Not used

File: README.md [hunk 5]
```diff
@@ -1063,6 +1066,34 @@ $mailersend->scheduleMessages->delete('message_id');
...
+$mailersend->token->getAll($page = 1, $limit = 10);
+```
+
+<a name="find-a-token"></a>
+
+### Find a token
+
+```php
+use MailerSend\MailerSend;
+use MailerSend\Helpers\Builder\TokenParams;
```
Comment (PeterOcansey) [inline README.md:1088]: Cleanup

**Review by PeterOcansey — COMMENTED** [review]
Just a few clean up

**Review by PeterOcansey — APPROVED** [review]

**Review by JoaoFSCruz — APPROVED** [review]

**Pull request merged**